corehq/apps/ota/views.py

```diff
@@ -181,9 +181,25 @@ def claim(request, domain):
 @require_POST
 @check_domain_migration
 def claim_all(request, domain):
-    username = request.POST.get("username")     # username may or may not be fully qualified
-    username = format_username(raw_username(username), domain)
-    user_id = username_to_user_id(username)
+    domain_obj = Domain.get_by_name(domain)
+    if not domain_obj:
+        return HttpResponse(_('Invalid project space "{}".').format(domain), status=500)
```

Comment:
I think should just return a 404.
```suggestion
        return HttpResponseNotFound(_('Invalid project space "{}".').format(domain))
```